.claude/scripts/log-to-langfuse-trace-example.py

```
@@ -0,0 +1,206 @@
+        }
+    )
+
+    span.end()  # end_time 기록 → duration 자동 계산
+
+    langfuse.flush()
+
+    # 세션 정리
+    clear_session()
+
+    print(f"✅ Green 커밋 Span 종료: {span.id}")
+    print(f"📊 LangFuse에서 TDD 사이클 시간 분석 가능")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)